packages/x-charts/src/LineChart/extremums.ts

```diff
@@ -21,21 +21,31 @@ export const getExtremumY: ExtremumGetter<'line'> = (params) => {
       (acc: ExtremumGetterResult, seriesId) => {
         const isArea = series[seriesId].area !== undefined;
 
-        const getValues = isArea
-          ? (d: [number, number]) => d
-          : (d: [number, number]) => [d[1], d[1]]; // Id area should go from bottom to top, without area should only consider the top
+        const getValues: (d: [number, number]) => [number, number] = isArea
+          ? (d) => d
+          : (d) => [d[1], d[1]]; // Since this series is not used to display an area, we do not consider the base (the d[0]).
 
-        const [seriesMin, seriesMax] = series[seriesId].stackedData.reduce(
+        const seriesExtremums: ExtremumGetterResult = series[seriesId].stackedData.reduce(
```

Comment:
You are accessing `series[seriesId].stackedData` 3 times
Maybes it's worth creating a variable for readability :+1: